src/sqlancer/duckdb/DuckDBOptions.java

```diff
@@ -77,6 +77,25 @@ public class DuckDBOptions implements DBMSSpecificOptions<DuckDBOracleFactory> {
     @Parameter(names = "--max-num-updates", description = "The maximum number of UPDATE statements that are issued for a database", arity = 1)
     public int maxNumUpdates = 5;
 
+    public enum CODDTestModel {
```

Comment:
If this is the same for all CODDTest implementations, perhaps we could extract this to a common enum with the next PR?